.rubocop.yml

```diff
@@ -0,0 +1,1158 @@
+  AllCops:
```

Comment:
I just want to be sure we don't loose our current style which should be based on github's one.